All EIPIP Meetings/Meeting 011.md

```diff
@@ -0,0 +1,552 @@
+
+**Pooja Ranjan**
+Yeah, sure.
+
+**James Hancock**
+It would be good to add the blog post to the resources.
+
+**Pooja Ranjan**
+Okay. So I have added the blog post here somewhere.  Okay, this was published by the Cat Herders. Okay. I think I missed one published by the Ethereum Foundation, I may have to add that blog post as well. So we try to provide as many links as possible to get into the real conversation and feel like we have not made up anything. Everything is coming up from the discussion. So yeah, I will add those at the bottom as well.
+
+**Chloethedev.eth**
+Yeah, for sure. I just know for me, sometimes, the EIPS can get very verbose and very long. And sometimes just scrolling to the bottom and seeing a few very important resources can be helpful. And so for other people that might get into other more technical post mortems, it might just be helpful to have those few key resources at the bottom, as well as all those lists that I'm sure people that are more into those post mortems will go through thoroughly.
+
+**Pooja Ranjan**
+Right. So one new thing that we are trying to do here is putting the suggested corrective action in the form of problem and suggestions. One of the comments that was received by one of the client developers was "The testing window planned for this particular upgrade was too short." Maybe because we did Istanbul in December and January it was there. So yeah, certainly, we did not have testing window, and maybe because of that, some miscalculation. The Ropsten had to be delayed for a very, very long time. It came up after 10 days or 11 days after the mainnet, and we should avoid mainnet to upgrade before testnet. So does anybody have any comment on this particular format, the problem in suggestion, or it looks fine, or can we put it in some other form?
+
+**unknown**
+I think it looks fine.
+
+**James Hancock**
+Great. It's been it's been fun. The last couple of weeks, Pooja and I have been  meeting and writing this out. It's long overdue. So I appreciate the help on making this happen. I think it's important to record this and so we can learn from it.
+
+**Pooja Ranjan**
+I'm going to stop sharing now, and we can move on to the next item unless anybody has any more question on it.
+
+**James Hancock**
+Anyone have thoughts on the the 2, the TV, or where to put the discussion link? Or what that should be like? The the discussion 2 usually goes to a post on the EIP, but in this case, thoughts on what's a good place to put it or if we should have one.
```

Comment:
Anyone have thoughts on the 2, the TV, or where to put the discussion link? Or what that should be like? The "discussion to" usually goes to a post on the EIP, but in this case, thoughts on what's a good place to put it or if we should have one.